### PR TITLE
1849 support ssl to mce cli.py

### PR DESCRIPTION
**Pull request by fabiofilz**
feat: Adding SSL support to mce_cli.py
Related to: https://github.com/linkedin/datahub/issues/1849

Test result:

MCE7: {"auditHeader": None, "proposedSnapshot": ("com.linkedin.pegasus2avro.metadata.snapshot.DataProcessSnapshot", {"urn": "urn:li:dataProcess:(sqoop,DEMO,PROD)", "aspects": [{"owners":[{"owner":"urn:li:corpUser:datahub","type":"DATAOWNER"}],"lastModified":{"time":1581407189000,"actor":"urn:li:corpUser:datahub"}},{"inputs":["urn:li:dataset:(urn:li:dataPlatform:cassandra,barEarth,DEV)","urn:li:dataset:(urn:li:dataPlatform:cassandra,barMars,DEV)"],"outputs":["urn:li:dataset:(urn:li:dataPlatform:hbase,barSky,PROD)","urn:li:dataset:(urn:li:dataPlatform:hbase,barOcean,PROD)"]}]}), "proposedDelta": None}

Flushing records...

### CODE REVIEW

File: metadata-ingestion/mce-cli/mce_cli.py [hunk 1]
```diff
@@ -78,11 +78,30 @@ def consume(conf, schema_record):
     c.close()
 
 
+def convert_config_to_dict(file_location):
+    #print("converting: " + file_location)
```
Comment (mars-lan) [inline metadata-ingestion/mce-cli/mce_cli.py:82]: Drop this line?

File: metadata-ingestion/mce-cli/mce_cli.py [hunk 2]
```diff
@@ -78,11 +78,30 @@ def consume(conf, schema_record):
...
+    result = {}
+    f = open(file_location, "r")
+    fl =f.readlines()
```
Comment (mars-lan) [inline metadata-ingestion/mce-cli/mce_cli.py:85]: Space after `=`

**Review by mars-lan — COMMENTED** [review]
Minor nits

File: metadata-ingestion/mce-cli/mce_cli.py (hunk 1 above)
Comment (fabiofilz) [reply metadata-ingestion/mce-cli/mce_cli.py]: Done

File: metadata-ingestion/mce-cli/mce_cli.py (hunk 2 above)
Comment (fabiofilz) [reply metadata-ingestion/mce-cli/mce_cli.py]: Done

**Review by mars-lan — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**